packages/flutter/test/material/dropdown_menu_test.dart

```diff
@@ -2128,6 +2128,50 @@ void main() {
     },
   );
 
+  // Regression test for https://github.com/flutter/flutter/issues/160196.
+  testWidgets(
+    'ListenableBuilder does not create infinite build loop with DropdownMenu',
+    (WidgetTester tester) async {
+      final TextEditingController controller = TextEditingController();
+      final FocusNode focusNode = FocusNode();
+      addTearDown(controller.dispose);
+      addTearDown(focusNode.dispose);
+
+      int buildCount = 0;
+      final Widget listenableBuilder = ListenableBuilder(
+        listenable: controller,
+        builder: (BuildContext context, Widget? child) {
+          buildCount += 1;
+          if (buildCount > 10) {
```

Comment:
I wonder, why exactly 10 as the limit here? Nothing else in the test seems to relate to this number, so could we lower this a little? Or does the infinite loop only trigger on the 11th rebuild?

Reply:
It was 100% arbitrary :)

Sure, I can lower it!